src/MonthView/MonthView.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react"
+import { Component } from "react"
+import { View } from "react-native"
+import { DayCalendarBox } from "./phone/DayCalendarBox"
+
+interface ComponentProps {}
+
+const weBusy = [
+    {name: "Adam", color: "blue"},
+    {name: "Adi", color: "green"},
+    {name: "Bruno", color: "red"},
+    {name: "George", color: "pink"},
+    {name: "Chiro", color: "black"},
+    {name: "Olivia", color: "grey"}
+]
+
+export class MonthView extends Component<ComponentProps, any> {
+
+    constructor(props: ComponentProps) {
+        super(props)
+        this.state = {
+            dateSelected: "",
+        }
+    }
+
+    public dayBoxOnPressHandler = (day: string) => {
```

Comment:
'day' in programming usually is 'monday - sunday'

can we rename this to 'date' so other people know what it means